app/Http/Controllers/EInvoicePeppolController.php

```diff
@@ -207,6 +208,40 @@ public function addAdditionalTaxIdentifier(AddTaxIdentifierRequest $request, Sto
         return response()->json(['message' => 'ok'], 200);
     }
 
+    public function removeAdditionalTaxIdentifier(RemoveTaxIdentifierRequest $request, Storecove $storecove): JsonResponse
+    {
+        /** @var \App\Models\Company $company */
+        $company = auth()->user()->company();
+        $tax_data = $company->tax_data;
+
+        $response = $storecove
+            ->proxy
+            ->setCompany($company)
+            ->removeAdditionalTaxIdentifier($request->validated());
+
+        if (data_get($response, 'status') === 'error') {
+            return response()->json(data_get($response, 'errors', 'message'), status: $response['code']);
+        }
+
+        if (is_bool($response)) {
+            // This means we run into "internal error" which means user request disconnect for VAT,
+            // that wasn't found on legal entity and Storecove network.
+
+            // This is place to do some logging, inform user, etc.
+        }
+
+        if ($request->country === 'GB') {
+            $tax_data->regions->UK->subregions->{$request->country}->vat_number = null;
+        } else {
+            $tax_data->regions->EU->subregions->{$request->country}->vat_number = null;
```

Comment:
Need to perform isset() checks on tax_data->regions->{country} as they may not be set